lizard_auth_server/templates/lizard_auth_server/profile.html

```diff
@@ -22,6 +23,34 @@ <h3>{% trans 'Your data' %}</h3>
           <dd>{{ view.profile.created_at }}</dd>
         </dl>
 
+      </div>
+      <div class="span4">
+
+        {% if view.oidc_userconsent %}
+          <h3>
+            {% trans 'Websites' %}
+          </h3>
+          <p>
+            {% blocktrans %}
+              Here are the websites you've given permission to log you in. You can delete this permission per site.
+            {% endblocktrans %}
+          </p>
+          <dl>
+            {% for oidc_userconsent in view.oidc_userconsent %}
+              <dt>{{ oidc_userconsent }}
+                  <td><a href="{% url 'lizard_auth_server.confirm_deletion_userconsent' pk=oidc_userconsent.id %}">
```

Comment:
Inconsistente indentatie. Twee spaties aanhouden.

Reply:
En als er meerdere items in zo'n "td" tag staan zet ik de begintag op z'n eigen regel (net zoals de eindtag op z'n eigen regel staat).

Reply:
Überhaupt: wat doet die TD hier? Binnen een DL vind je alleen DT en DD :-)

Reply:
Done